runtime/src/common/sgx/avr.rs

```diff
@@ -100,12 +100,12 @@ lazy_static! {
     static ref IAS_TRUST_ANCHOR: Vec<u8> = {
         let pem = match parse_x509_pem(IAS_TRUST_ANCHOR_PEM.as_bytes()) {
             Ok((rem, pem)) => {
-                if !rem.is_empty() {
-                    panic!("anchor PEM has trailing garbage");
-                }
-                if pem.label != PEM_CERTIFICATE_LABEL {
-                    panic!("PEM does not contain a certificate: '{:?}'", pem.label);
-                }
+                assert!(rem.is_empty(), "anchor PEM has trailing garbage");
```

Comment:
Are asserts in rust unconditionally compiled?

Reply:
ya I had to look this up, and `debug_assert!` is the conditional one